src/pages/Items/ProductDetail/ProductDetailInquiry.tsx

```diff
@@ -0,0 +1,35 @@
+import { ChangeEvent } from "react";
+
+interface ProductDetailInquiryProps {
+  inquiryValue: string;
+  setInquiryValue: (value: string) => void;
+}
+
+const ProductDetailInquiry = ({
+  inquiryValue,
+  setInquiryValue,
+}: ProductDetailInquiryProps) => {
```

Comment:
개별 파일에서 컴포넌트의 타입은 굳이 컴포넌트명(ProductDetailInquiry) + 프롭스(Props)로 네이밍하기 보다는 그냥 Props 정도로 네이밍하는 경우가 많습니다. 왜냐하면 다른 타입과 달리 Prop는 컴포넌트에 하나만 존재할 수 있기 때문에, 별도의 prefix 없이도 보는 사람이 비교적 잘 인지할 수 있어서요. 향후 코드 작성 시 참고하시면 좋을 것 같습니다. 😁

```suggestion
}: Props) => {
```